e2e/src/step_definitions/media-viewer.ts

```diff
@@ -122,11 +128,29 @@ const addComment = async (comment: string) => {
   await page.clickOnSaveButton();
 };
 
-const highLightTextInPdf = async () => {
+const highLightTextInPdf = async function () {
+   await page.waitForPdfToLoad();
+   await sleep(5000);
+   await toolBar.enableTextHighLightMode();
+   await page.highLightTextOnPdfPage();
+};
+
+const highLightTextForBookmarking = async function () {
   await page.waitForPdfToLoad();
-  await sleep(5000);
   await toolBar.enableTextHighLightMode();
-  await page.highLightTextOnPdfPage();
+  await page.highLightTextForBookmarking();
+};
+
+const loadPdfForBookmarking = async function() {
```

Comment:
this doesn't seem to do anything apart from sleeping

Reply:
removed from code.